server/db/indexpsql.js

```diff
@@ -0,0 +1,30 @@
+const { Pool } = require('pg');
+
+const pool = new Pool({
+  host: 'localhost',
+  user: 'student',
+  password: 'student',
+  database: 'gallerymodule',
+});
```

Comment:
Are you able to connect without indicating a port?

Reply:
Yeah, I believe I only need the port number for my Express server. I've had problems explicitly defining a port number